docs/Salesforce-batchsource.md

```diff
@@ -33,6 +33,12 @@ A Salesforce connected application must be created in order to get a client secr
 
 **SOQL Query:** A SOQL query to fetch data into source.
 
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
+
+**Connection:** Name of the connection to use. Object Names information will be provided by the connection.
+You also can use the macro function ${conn(connection-name)}.
+
```

Comment:
Same problem as with the Salesforce sink .md. SObject Name seems to have been removed and the example is under the Connection property.

Reply:
Changed the sequence.